main.fmf

```diff
@@ -5,5 +5,5 @@ require:
   - type: file
     pattern: /TestHelpers/.*
   - url: https://github.com/RedHat-SP-Security/common-cloud-orchestration
-    ref: main
+    ref: pk_iib_parsed_image
```

Comment:
Is this really intended?

Reply:
No, before merge I'll set it to main, this is just for testing purposes.